twine/generate/triggers.c

```diff
@@ -153,11 +163,17 @@ int
 spindle_triggers_index(SQL *sql, const char *id, SPINDLEENTRY *data)
 {
 	size_t c;
+	char nextactivationstr[32];
+	time_t now;
+	struct tm tm;
 
 	for(c = 0; c < data->ntriggers; c++)
 	{
-		if(sql_executef(sql, "INSERT INTO \"triggers\" (\"id\", \"uri\", \"flags\", \"triggerid\""") VALUES (%Q, %Q, '%d', %Q)",
-			id, data->triggers[c].uri, data->triggers[c].kind, data->triggers[c].id))
+		now = time(NULL);
+		gmtime_r(&now, &tm);
+		strftime(nextactivationstr, 32, "%Y-%m-%d %H:%M:%S", &tm);
```

Comment:
I think it's ok to set it to NOW() by default. Could actually probably best to that in SQL directly...